src/main/java/seedu/address/commons/util/StringUtil.java

```diff
@@ -65,4 +65,22 @@ public static boolean isNonZeroUnsignedInteger(String s) {
             return false;
         }
     }
+
+    /**
+     * Trims s and removes the trailing zero (.0) of s, if any.
+     *
+     * @param s The string to remove the trailing zero from.
+     * @return Trimmed string without trailing zero.
+     */
+    public static String removeTrailingZero(String s) {
+        requireNonNull(s);
+
+        String trimmedString = s.trim();
+
+        if (trimmedString.endsWith(".0")) {
+            return trimmedString.substring(0, trimmedString.length() - 2);
```

Comment:
would the -2 be considered a magic literal?

Is it better to convert it to a double, then a int so it will automatically truncate the decimal point? Something like this?
 double doubleValue = Double.parseDouble(s.trim());
 int intValue = (int) doubleValue;
 return Integer.toString(intValue);